js/base.js

```diff
@@ -60,8 +60,14 @@ export class API extends mix(BaseAPI).with(DocumentAPI, ShipmentAPI) {
     }
 
     async _handleResponse(response, errorMessage = "Problem in request") {
-        const result = await super._handleResponse(response, errorMessage);
+        const result = await response.json();
+        const errors =
+            result.response && result.response.errors
+                ? result.response.errors.map(JSON.stringify).join(",")

```

Comment:
I'd also prefer this, but we don't use function references
```suggestion
                ? result.response.errors.map(error => JSON.stringify(error)).join(",")

```